Dikubot/DataLayer/Database/Global/Facebook/FacebookPageService.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Dikubot.DataLayer.Database.Global.Facebook;
+
+public class FacebookPageService : GlobalMongoService<FacebookPage>
```

Comment:
See the naming conventions we have used for database services. Consider changing it to FacebookPageDatabaseService